### ISSUE TITLE
"SSH connect inside containers" doesn't work

### ISSUE THREAD

**Issue by sotetsuk** [Microsoft/pai#1576]
Hi, let me report a bug on `v0.8.0`.

The feature "[SSH connect inside containers](https://github.com/Microsoft/pai/blob/master/docs/job_tutorial.md#ssh-connect-inside-containers)" looks like being broken on `v0.8.0`.

The errors are like:

```sh
# ssh -oStrictHostKeyChecking=no <TASK_NAME>   
Warning: Permanently added '[XX.XXX.XX.XXX]:YYYYY' (ECDSA) to the list of known hosts.
no such identity: /root/.ssh/application_XXXXXXXXXX: No such file or directory
```

The `.ssh` dir is like
 
```sh
~/.ssh # ls
authorized_keys  
config  
JOB_NAME.pub
```

In a previous version, the `.ssh` dir is like

```sh
~/.ssh # ls
authorized_keys
config
application_XXXXX
application_XXXXX.pub
```

So I feel like that this error comes from these two reasons:

1. The item `Identityfile` in `.ssh/config` is wrongly set, specifying the old name like `appllication_XXXXX`
2. No secret key is set under `.ssh`

Thanks,

**Comment by wangcan0329** [comment]
Hi, in v0.8.0 we changed the ssh key file name from ${application_id} to ${Job_Name} and we won't download the private key file to each container automatically. we will update the document and re-enable SSH connect inside containers.

**Comment by qyyy** [comment]
This issue probably is the reason why mpi example cannot run correctly. And to support horovod, this feature is also needed.

**Comment by fanyangCS** [comment]
here is the fix: https://github.com/Microsoft/pai/pull/1582, please wait for redeployment.

**Comment by wangcan0329** [comment]
Already merged the PR [#1582](https://github.com/Microsoft/pai/pull/1582) , please wait for prod deployment. I will close this issue first. Thanks!
